nixnet_examples/generic_synchronization.py

```diff
@@ -25,7 +25,7 @@ def main():
     synchronized_input.intf.can_term = constants.CanTerm.ON
 
     #Connect the start trigger terminals for the synchronized interface
-    synchronized_input.connect_terminals(nixnet._cconsts.NX_TERM_FRONT_PANEL0, nixnet._cconsts.NX_TERM_START_TRIGGER)
+    synchronized_input.connect_terminals(constants.Terminals.NX_TERM_FRONTPANEL0, constants.Terminals.NXTERM_STARTTRIGGER)
```

Comment:
Don't you need to modify these functions
1. To convert from enum to numeric value
2. To document the change in data type?